kubernetes/kubernetes_service.go

```diff
@@ -236,3 +267,21 @@ func (in *IstioClient) getDeployments(namespace string, deploymentsChan chan dep
 	deployments, err := in.k8s.AppsV1beta1().Deployments(namespace).List(emptyListOptions)
 	deploymentsChan <- deploymentsResponse{deployments: deployments, err: err}
 }
+
+func (in *IstioClient) getDeployment(namespace string, deploymentName string) (*v1beta1.Deployment, error) {
```

Comment:
I think we already have it: exactly the same except first letter cap: `GetDeployment(namespace, deploymentName string)`